### DISCUSSION TITLE
why CefSharp disable GPU can reduce CPU usage？

### DISCUSSION THREAD

**Discussion by liangyuqi** [cefsharp/CefSharp#4908] [General]
[code](https://github.com/cefsharp/CefSharp/blob/master/CefSharp.Example/CefExample.cs#L103)

```
 // NOTE: For OSR best performance you should run with GPU disabled:
 // `--disable-gpu --disable-gpu-compositing --enable-begin-frame-scheduling`
 // (you'll loose WebGL support but gain increased FPS and reduced CPU usage).
 // http://magpcss.org/ceforum/viewtopic.php?f=6&t=13271#p27075
 //https://bitbucket.org/chromiumembedded/cef/commits/e3c1d8632eb43c1c2793d71639f3f5695696a5e8
 
 ```
why “you'll loose WebGL support but gain increased FPS and reduced **CPU** usage”

i think it will reduce **GPU** usage，its a typo？

**Comment by liangyuqi** [comment]
and i have to use webgl，but when i enable-gpu，the website interaction becomes very laggy。what should i do

**Comment by amaitland** [comment]
> http://magpcss.org/ceforum/viewtopic.php?f=6&t=13271#p27075

Read the linked post.

Are you using OSR?

The upstream example has changed slightly https://github.com/chromiumembedded/cef/blob/2cd405baac5cbef7c5d4583be14ce64a3d6eed9f/tests/shared/browser/client_app_browser.cc#L53

CefSharp should update the examples to match.

If you are having performance issues then test with https://github.com/cefsharp/CefSharp.MinimalExample for a comparison.

Low level CEF questions should be asked on https://www.magpcss.org/ceforum/index.php
